TestsFoundation/TestsFoundation/DataSeeding/Course/CreateDSCourseRequest.swift

```diff
@@ -40,5 +40,35 @@ extension CreateDSCourseRequest {
     public struct Body: Encodable {
         let course: RequestedDSCourse
         let offer = true // makes the course published after creation
+
+        public init(course: RequestedDSCourse) {
+            self.course = course
+        }
+    }
+}
+
+struct UpdateDSCourseRequest: APIRequestable {
```

Comment:
If these are about updates would it make sense to create an `UpdateDScourseRequest.swift` file for them?